tests/utils.js

```diff
@@ -492,6 +492,13 @@ define([
                 }
                 mug.p[name] = val;
             });
+
+            // Open all sections
```

Comment:
Yeah...I see your point, but I am also not eager to change all of those tests. Looking towards the future, if we change the default sections again, tests that fail because some element wasn't visible can be pretty obnoxious to track down.